lighthouse/constants/fields.py

```diff
@@ -27,6 +27,15 @@
 # source_plates collection
 FIELD_BARCODE = "barcode"
 
+# events collection
+FIELD_EVENT_RUN_ID = "automation_system_run_id"
```

Comment:
I would keep the collection name the same in the assignment.
```suggestion
FIELD_EVENTS_RUN_ID = "automation_system_run_id"
```

Reply:
I think I dont understand this change, why is plural?